### ISSUE TITLE
Reference Fields Not Implemented

### ISSUE THREAD

**Issue by rooflack** [MiniKeePass/MiniKeePass#507]
It looks like special values like "{REF:...}", that in the desktop app are used to refer to values stored in other fields or entries, are treated like regular literal values in MiniKeePass and not de-referenced when displayed or copied to clipboard.
I'm not sure how often this feature is used, but it would be a nice addition to the mobile app.
Thanks!
GM

**Comment by shad0whawk** [comment]
I agree it would be useful to have this added since I use it as welll.

**Comment by metbril** [comment]
Duplicate of #204.
